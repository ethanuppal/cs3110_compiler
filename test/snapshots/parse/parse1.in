// this is syntactically incorrect at line 2, col 1
)
