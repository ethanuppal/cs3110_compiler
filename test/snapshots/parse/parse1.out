Syntax error at parse1.in:2:1
